docker/Dockerfile-postgres

```diff
@@ -0,0 +1,22 @@
+FROM python:3.6-alpine
+
+COPY requirements*.txt /tmp/
+
+RUN apk update && apk upgrade
+
+RUN apk add --no-cache --virtual .build-deps gcc musl-dev make postgresql-dev && \
+    apk add --update --no-cache postgresql &&\
+    pip install --disable-pip-version-check --no-cache-dir -r /tmp/requirements.txt \
+    -r /tmp/requirements-postgres.txt && \
+    apk del .build-deps && \
+    rm /tmp/requirements*.txt && \
+    mkdir /app && \
+    chown 1001:1001 /app
```

Comment:
this ownership setup has no apparent purpose for this image, but let's maybe keep it for now and for the sake of uniformity